src/Redux/books/books.js

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+
+const Booksec = () => (
+  <>
+    <div className="books-container d-flex">
+      <h2>Book List</h2>
+      <button type="button">REMOVE</button>
+    </div>
+    <input type="text" name="addBooks" placeholder="Enter Book Title" />
+    <button type="button">Add Book</button>
```

Comment:
- [ ] As it is a requirement for this project, please add `CreateNewBook` component that renders a form with inputs for the title and the author of the book, has a button to submit(the button wouldn't add the book yet, this is just a placeholder for now).

Reply:
Hello Mihndim2020,
The project requirement is about the initialization of react app and redux, I think what you are saying is for the next project
Thank you.